projects/vdk-plugins/vdk-ipython/README.md

```diff
@@ -14,19 +14,21 @@ Then to load the extension in Jupyter the user should use:
 ```
 %reload_ext vdk_ipython
 ```
-And to load the job_input:
+And to load the data_job (Job Control object):
 ```
-%reload_job_input
+%reload_data_job
 ```
-The %reload_job_input magic can be used with arguments such as passing the job's path with --path
+The %reload_data_job magic can be used with arguments such as passing the job's path with --path
 or giving the job a new with --name, etc.
 
 ### Example
 The output of this example is "myjob"
 ```
 %reload_ext vdk_ipython
 
-%reload_job_input --name=myjob
+%reload_data_job --name=myjob
+
+job_input = data_job.get_initialized_job_input()
```

Comment:
Does it make a difference for the end user to be utilising `get_initialized_job_input` or simply `get_job_input`? Is "initialized" meaningful to keep in mind from user perspective?

Reply:
I left it like because of the finalise method, because I thought that users might think "What did I initialise to have a finalization" or sth like that. When we are ready with the finalise method, we can change the naming